webapi/src/main/scala/org/knora/webapi/messages/admin/responder/permissionsmessages/PermissionsMessagesADM.scala

```diff
@@ -23,6 +23,23 @@ import java.util.UUID
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 // API requests
 
+/**
+ * Validates permission IRI
+ * @param iri to be validated.
+ */
+private case class validatePermissionIRI(iri: IRI) {
```

Comment:
class names should be upper case

Reply:
Obsolete, I did some refactoring just before a while.